.gitignore

```diff
@@ -0,0 +1,66 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Keep such files to yourself. Add them to your global gitignore. These are not meant to be added to pull requests.